spi/common/identity-did-spi/src/main/java/org/eclipse/edc/iam/did/spi/document/DidDocument.java

```diff
@@ -28,7 +28,7 @@
 @JsonDeserialize(builder = DidDocument.Builder.class)
 public class DidDocument {
 
-    public static final String DID_RESOLUTION_CONTEXT = "https://w3id.org/did-resolution/v1";
+    public static final String DID_RESOLUTION_CONTEXT = "https://www.w3.org/ns/did/v1";
```

Comment:
```suggestion
    public static final String DID_CONTEXT = "https://www.w3.org/ns/did/v1";
```
Just a nit but i would change also the constant name